includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -231,26 +239,34 @@ public function getNavigation() {
 	 */
 	public function getAllElements() {
 		$params = $this->getRequestParameters();
+		$footerModel = new DesignSystemGlobalFooterModel(
+			$params[self::PARAM_PRODUCT],
+			$params[self::PARAM_ID],
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)